src/main/java/uk/gov/hmcts/reform/sscs/services/mapper/TransformAppealCaseToCaseData.java

```diff
@@ -30,19 +42,23 @@
     public SscsCaseData transform(final AppealCase appealCase) {
         List<Parties> parties = appealCase.getParties();
 
-        Optional<Parties> appellantParty = (parties == null) ? Optional.empty() :
-            parties.stream().filter(f -> f.getRoleId() == APPELLANT_ROLE_ID).findFirst();
-
-        if (!appellantParty.isPresent()) {
-            log.error("An appeal, for caseId {}, exists without an appellant. This cannot be possible.",
-                appealCase.getAppealCaseId());
+        Optional<Parties> appointeeParty = (parties == null) ? Optional.empty() :
+            parties.stream().filter(f -> f.getRoleId() == APPOINTEE_ROLE_ID).findFirst();
+        Optional<Parties> appellantParty;
+        if (!appointeeParty.isPresent()) {
+            appellantParty = (parties == null) ? Optional.empty() :
+                parties.stream().filter(f -> f.getRoleId() == APPELLANT_ROLE_ID).findFirst();
```

Comment:
Duplicate of APPELLANT check, please create function/method

Reply:
done